homeassistant/components/system_bridge/sensor.py

```diff
@@ -240,7 +245,7 @@ async def async_setup_entry(
             SystemBridgeSensor(coordinator, description, entry.data[CONF_PORT])
         )
 
-    for key, _ in coordinator.data["disk"].items():
+    for key, _ in coordinator.data.disk.__dict__.items():
         if "_percent" in key.lower():
             partition = key.replace("usage_", "").replace("_percent", "")
```

Comment:
If the partition is generally useful for library users maybe it should be an attribute of the model returned by the library?

Reply:
Problem with this is the name is dynamic so it could be C:\ or /home etc.

Reply:
I see what you mean. I'll add a list of partitions to the model/module as well as replacing others like this